torch/quantization/observer.py

```diff
@@ -264,6 +293,22 @@ def calculate_qparams(self):
     def extra_repr(self):
         return "min_val={}, max_val={}".format(self.min_vals, self.max_vals)
 
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super(PerChannelMinMaxObserver, self)._save_to_state_dict(destination, prefix, keep_vars)
+        destination[prefix + 'min_vals'] = self.min_vals
```

Comment:
can't they just be buffers and get automatic serialization?

Reply:
Yes, for the per-channel case that can be done.